README.md

```diff
@@ -1,75 +1,103 @@
-# Determined: Deep Learning Training Platform
-
 <p align="center"><img src="determined-logo.png" alt="Determined AI Logo"></p>
 
-Determined is an open-source deep learning training platform that makes building
-models fast and easy. Determined enables you to:
+Determined is an all-in-one deep learning platform, compatible with PyTorch and Tensorflow.
```

Comment:
Determined is an all-in-one deep learning platform, compatible with PyTorch and TensorFlow.